src/DIRAC/Core/Utilities/JEncode.py

```diff
@@ -107,7 +108,7 @@ def default(self, obj):  # pylint: disable=method-hidden
             return obj._toJSON()  # pylint: disable=protected-access
         # if the object a bytes, decode it
         elif isinstance(obj, bytes):
-            return obj.decode()
+            return {"__dCls": "b64", "obj": b64encode(obj).decode()}
```

Comment:
Can you update the encoding tests to include one that gets hypothesis to test using `bytes`

Reply:
done in #5811